great-idea/css/index.css

```diff
@@ -69,76 +69,71 @@ h1, h2, h3, h4, h5 {
 	border: 1px solid black;
 	width: 888px;
 	margin: 0px auto;
-	vertical-align: center;
 	display: flex;
 	flex-direction: column;
-	align: center;
-
+	
 }
 
 	.container-header {
-		border: 4px solid red;
-		/* width: 880px; */
-		height: 250px;
-		margin: 0px 10px 0px 10px;
+		/* border: 4px solid red; */
+		height: 280px;
 		display: flex;
 		flex-direction: column;
 		justify:center;
-		/* z-index: 1; */
-		/* justify-content: space-around; */
 	}
 		
 		.box-nav {
-			border: 1px solid red;
-			text-decoration: none;
-			width: 95%;
-			height: 20px; 
+			/* border: 3px solid yellow; */
+			/* text-decoration: none; */
+			width: 100%;
+			height: 50px; 
 			display: flex;
-			justify-content: flex-start;
+			justify-content: space-between;
 
 		}
 		
-		/* .box-logo {
-			border: 1px solid red;
-			display: flex;
-			width: 49%;
-			justify-content: flex-end;
-		} */
-
 		.box-services-header {
-			border: 10px solid green;
-			/* display: flex; */
-			
-			
+			/* border: 3px solid green; */
+			width: 880px;
+			display: flex;
 		}
+			.services-header-img {
+				width: 100%;
+			}
 	
 	.container-services {
 		/* border: 3px solid purple; */
-		margin: 10px;
+		height: 120px;
+		margin: 5px;
+		border-bottom: 2px solid black;
 	}
 		.box-services {
-			border: 1px solid red;
+			/* border: 1px solid red; */
 			flex-direction: column;
 			display: flex;
 			justify-content: space-around;
-			border-bottom: 2px solid black;
+			
 		}
 
 	.container-main {
-		border: 3px solid blue;
+		/* border: 3px solid blue; */
 		height: 600px;
 		display: flex;
 		flex-wrap: wrap;
-		flex-direction: column;
+		flex-direction: row;
 		justify:center;
-		justify-content: space-around;
+		align: center;
+		border-bottom: 2px solid black;
+		padding-bottom: 3px;
+		/* justify-content: space-around; */
 	}
 		.box-digital-design {
-			border: 2px solid black;
-			width: 350px;
-			padding: 7px;
-			flex-wrap: wrap;
-			align-self: center
+			border: 1px solid black;
+			width: 49%;
+			height: 33% auto;
```

Comment:
height: 33% auto is not a valid property value. Height should almost never have a percentage value and auto is something you would use when you are resizing an image. Say you have an image that is 100px wide and 200px tall. If you changed the width property to 150px. To maintain the aspect ratio you would either have to change the height property to 300px or set it to auto.